cmd/fossa/report.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	logging "github.com/op/go-logging"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/config"
+)
+
+var reportLogger = logging.MustGetLogger("report")
+
+type dependencyResponse struct {
+	Loc struct {
+		Package  string
+		Revision string
+	}
+	Licenses []licenseResponse
+	Project  struct {
+		Title   string
+		URL     string
+		Authors []string
+	}
+}
+
+type licenseResponse struct {
+	ID       string
```

Comment:
should include SPDX id; we should prefer using SPDX in general

Reply:
Fixed.

Reply:
Will fix.